<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="IN-PG02H8SH" failures="2" tests="2" name="Test(failed)" time="0.079" errors="0" timestamp="2022-06-26T16:00:10 IST">
  <testcase classname="Test.LoginTest" name="login" time="0.01">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Object.LoginPage.clickonSignIn(LoginPage.java:20)
at Test.LoginTest.login(LoginTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="Test.HomeTest" name="init" time="0.002">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Object.HomePage.clickOnSearch(HomePage.java:16)
at Test.HomeTest.init(HomeTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- init -->
  <testcase classname="Object.Base" name="@BeforeSuite initialize" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite initialize -->
  <testcase classname="Object.Base" name="@AfterSuite TeardownTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite TeardownTest -->
</testsuite> <!-- Test(failed) -->
